tests/usbredir_test.go

```diff
@@ -23,6 +23,8 @@ import (
 	"io"
 	"time"
 
+	"kubevirt.io/kubevirt/tests/libvmi"
```

Comment:
nit: nicer under `kubevirt.io/kubevirt/tests`

Reply:
make generate keep locating this here for some reason :\

Reply:
it is creating it there; but if you move it elsewhere I bet it would not object. It's a bit simpler for humans to read when it sits among related files.

Reply:
I think that there is a lane that fails if `make generate` change files but let's try this